lib/matplotlib/_mathtext.py

```diff
@@ -2599,3 +2603,30 @@ def auto_delim(self, s, loc, toks):
             # if "mid" in toks ... can be removed when requiring pyparsing 3.
             toks["mid"].asList() if "mid" in toks else [],
             toks["right"])
+
+    def boldsymbol(self, s, loc, toks):
+        self.push_state()
+        state = self.get_state()
+        lw = list(map(chr, range(ord('a'), ord('z') + 1)))
+        up = list(map(chr, range(ord('A'), ord('Z') + 1)))
+        smGreek = list(map(lambda a: unicodedata.name(chr(a))
+                           .split()[-1]
+                           .lower(),
+                           range(ord('\N{GREEK SMALL LETTER ALPHA}'),
+                                 ord('\N{GREEK SMALL LETTER OMEGA}') + 1)))
+        allLatin = [*lw, *up]
+        hlist = []
+        name = toks["value"]
+        for c in name:
+            if isinstance(c, Char):
+                c.font = "bf"
+                if str(c)[1] in allLatin or str(c)[2:-1] in smGreek:
```

Comment:
`str(c)[1:-1]` is equal to `c.c` for `Char`, I think?
```suggestion
if c.c in string.ascii_letters or c.c[1:] in lower_greek:
```